test/specs/chart/LineChartSpec.js

```diff
@@ -508,3 +508,203 @@ describe('<LineChart /> - Pure Rendering with legend', () => {
   });
 
 });
+
+describe("<LineChart /> - Rendering two line charts with syncId", () => {
+  const margin = { top: 20, right: 20, bottom: 20, left: 20 };
+  const height = 400;
+  const width = 400;
+
+  // Reversed of data
+  const data2 = [
```

Comment:
Same comment as above (having a different set of `uv` values

Reply:
done